app/controllers/replies_controller.rb

```diff
@@ -116,7 +117,24 @@ def create
       redirect_to post_path(post_id, page: :unread, anchor: :unread) and return
     elsif params[:button_preview]
       draft = make_draft
-      preview(ReplyDraft.reply_from_draft(draft)) and return
+      preview_reply(ReplyDraft.reply_from_draft(draft)) and return
+    elsif params[:button_submit_previewed_multi_reply]
+      if editing_multi_reply?
+        edit_reply(true)
+      else
+        post_replies
+      end
+      return
+    elsif params[:button_discard_multi_reply]
+      flash[:success] = "Replies discarded."
+      if editing_multi_reply?
+        # Editing multi reply, going to redirect back to the reply I'm editing
+        redirect_to reply_path(@reply, anchor: "reply-#{@reply.id}")
+      else
+        # Posting a new multi reply, go back to unread
+        redirect_to post_path(params[:reply][:post_id], page: :unread, anchor: :unread)
```

Comment:
Wait why do you have this on the discard path

Reply:
If you discard the multi replies from writing new ones you go to unread, but if you discard them from _editing_ one into multiple you go to the one you were editing.

Reply:
Aha. The comment confused me, but that makes sense.